docs/vspec2id.md

```diff
@@ -43,6 +45,9 @@ cd path/to/your/vss-tools
 Great, you generated your first overlay that will also be used as your validation file as soon as you update your
 vehicle signal specification file.
 
+If needed you can make the static UID generation case-sensitive using the command line argument `--strict-mode`. It
+will default to false.
```

Comment:
What does case sensitive mean here? That two signals `A.B` and `A.b` get different UID so that renaming  `A.b` to `A.B` is considered a name change?